js/search/searchLibrary.js

```diff
@@ -13,6 +13,25 @@ const symLucyIndexerPtr = ref.refType(symLucyIndexer);
  * using the node-ffi
  */
 let libSymphonySearch = ffi.Library(searchConfig.LIBRARY_CONSTANTS.SEARCH_LIBRARY_PATH, {
+
+    //New Memory Indexing API
```

Comment:
Should be `// New Memory Indexing API`. Please mind the linting before push a commit.